src/states/common/approved.rs

```diff
@@ -142,6 +143,14 @@ pub trait Approved: Relocated {
                     related_info,
                 } => {
                     // FIXME: Validate with Chain info.
+
+                    trace!(
```

Comment:
Is this trace log a left over from the debug work, or we do need this info somehow?

Reply:
It was useful for debugging and match the other trace logs in the same function.